src/app/weather-service.service.ts

```diff
@@ -0,0 +1,29 @@
+import {Injectable, Input} from '@angular/core';
+import {HttpClient, HttpHeaders} from '@angular/common/http';
+import {WeatherDataComponent} from './weather-data/weather-data.component';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WeatherServiceService {
+  locationLatLon = [`latitude=53.104709519168686&longitude=-3.9125189914657086`,`latitude=53.31727762086625&longitude=-3.481558103099215`,`latitude=52.13176376364786&longitude=-4.546721048689262`];
```

Comment:
Just going to query having lat/lons to 15 decimal places. Do they need to be that level of precision?

I refer you to this classic: https://xkcd.com/2170/